var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -73,6 +74,11 @@ class Trilinos(CMakePackage):
             description='global ordinal type for Tpetra')
     variant('fortran',      default=True,
             description='Compile with Fortran support')
+    variant('wrapper', default=False,
+            description="Use nvcc-wrapper for CUDA build")
+    variant('std', default='11', values=['11', '14', '17'], multi=False)
```

Comment:
Sorry, I should have caught this earlier, but the standard Spack variant for C++ standard is `cxxstd` (`"variant('.*std'" */package.py`). Could you change it and line 733?

Reply:
Fixed now